package.json

```diff
@@ -55,7 +55,8 @@
     "test:hub": "NODE_ENV=test cd packages/hub && yarn test",
     "test:did-resolver": "cd packages/did-resolver && yarn test",
     "test:discord-bot": "cd packages/discord-bot && yarn test",
-    "test:web-client": "cd packages/web-client && ember test",
+    "test:web-client": "cd packages/web-client && percy exec -- ember test",
+    "test:web-client:percy": "cd packages/web-client && ember test",
```

Comment:
Did you mean this the other way around? It doesn't look like the percy build ran because of the mixup

Reply:
lol oops, fixed now, thank you